Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1692,7 +1692,12 @@ def delete_context_table_records(client: Client, args: Dict) -> Tuple[Any, Dict[
     session_id = args.get('session_id')
     records = argToList(args.get('records'))
 
-    record_updates_raw_data = client.delete_context_table_records_request(context_table_name, records, session_id)
+    records_raw_data = client.list_context_table_records_request(context_table_name, 10000, 1)
```

Comment:
why aren't you using the `session_id` and `records` arguments?

Reply:
they're not used in the list_context_records_request function is the first place, they're used in the delete one